packages/litexa/src/parser/lambda-db-dynamo.coffee

```diff
@@ -0,0 +1,117 @@
+###
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+# ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+###
+
+dynamoDocClient = new AWS.DynamoDB.DocumentClient {
+  convertEmptyValues: true
+  service: new AWS.DynamoDB {
+      maxRetries: 5
+      retryDelayOptions:
+        base: 150
+      paramValidation: false
+      httpOptions:
+        agent: new (require 'https').Agent({ keepAlive: true })
+    }
+}
+
+db =
+  fetchDB: ({ identity, dbKey, fetchCallback }) ->
+    ttlConfiguration = litexa.ttlConfiguration
```

Comment:
making sure, `litexa` exists when this is called?

Reply:
yup, it's always defined in the injected preamble during toLambda, so it should be visible here. As it's (theoretically at least) a static value, it should be fine to read it directly off the global for this case.